modular_skyrat/modules/deforest_medical_items/code/vulnerable_status_effect.dm

```diff
@@ -0,0 +1,28 @@
+/atom/movable/screen/alert/status_effect/vulnerable_to_damage
+	name = "Vulnerable To Damage"
+	desc = "You will take more damage than normal while your body recovers from mending itself!"
+	icon_state = "terrified"
+
+/datum/status_effect/vulnerable_to_damage
+	id = "vulnerable_to_damage"
+	duration = 5 MINUTES
+	alert_type = /atom/movable/screen/alert/status_effect/vulnerable_to_damage
+	remove_on_fullheal = TRUE
+	/// The percentage damage modifier we give the mob we're applied to
+	var/damage_resistance_subtraction = 50
+	/// How much extra bleeding the mob is given
+	var/bleed_modifier_addition = 1
+
+/datum/status_effect/vulnerable_to_damage/on_apply()
+	to_chat(owner, span_userdanger("Your body suddenly feals weak and fragile!"))
+	var/mob/living/carbon/human/carbon_owner = owner
+	carbon_owner.physiology.damage_resistance -= damage_resistance_subtraction
```

Comment:
Something about this _really_ doesn't look right to me, shouldn't it be a multiplier instead, so we avoid divisions by zero? Or am I going to be very sad learning that's how it's usually done elsewhere in the codebase?

Reply:
That is, unfortunately, how it is done everywhere. Damage resistance is a percentage that is added to and taken from by a lot of things and multiplication has potential to permanently change that value when combined with certain other things.

Reply:
That's so sad, AI play Cargonia.